src/model/usuario.js

```diff
@@ -0,0 +1,37 @@
+import {ConjuntoGeneros} from "./conjunto_generos.js"
+
+
+/**
+ * @class Usuario
+ * Asocia a un usuario con sus gustos (sobre géneros de los libros)
+ */
+export class Usuario{
+   #gustos
+
+   /**
+    * @constructor
+    * Crea un usuario que tiene gustos referidos a géneros
+    * @param {ConjuntoGeneros} gustos - Conjunto de gustos
+    */
+   constructor(gustos){
+      this.#gustos = gustos
+   }
+
+   
+   /**
+    * @returns {Set} Devuelve un set que es el conjunto de gustos/géneros del usuario
+    */
+   get gustos(){
+      return this.#gustos.generos
+   }
+
+
+   /**
+    * Cambia los gustos del usuario
+    * @param {ConjuntoGeneros} nuevosGustos
+    */
+   set gustos(nuevosGustos){
```

Comment:
Primero, en este milestone se trata solo de modelizar, no de añadir funcionalidad. Por otro lado, repetidamente os he dicho en clase que *no* pongáis setters. El API de una clase debe regular cómo muta, y si lo hace o no. Si es mutable, ya no es un objeto valor, por otro lado, pero en todo caso, tendrá que mutar solamente de las formas que permita el interfaz.
Todo esto es consecuencia de que has hecho el cambio en 670e03ced7b6c6c184b9b5b5f3b7d82a0cde7ee0 que no apunta a ningún issue, *y por tanto es incorrecto*.